src/Servers/HttpSys/src/UrlPrefixCollection.cs

```diff
@@ -15,6 +19,16 @@ public class UrlPrefixCollection : ICollection<UrlPrefix>
         private UrlGroup _urlGroup;
         private int _nextId = 1;
 
+        // Valid port range of 5000 - 48000.
+        private const int BasePort = 5000;
+        private const int MaxPortIndex = 43000;
+        private const int MaxRetries = 1000;
+        private static int NextPortIndex;
+
+        private const int ErrorAccessDenied = 5;
+        private const int ErrorSharingViolation = 32;
+        private const int ErrorAlreadyRegistered = 183;
```

Comment:
Use/update the ErrorCodes class rather than define these locally.
https://github.com/aspnet/AspNetCore/blob/71c5c66b211c7ab9d44751e39cef27c525124bb7/src/Shared/HttpSys/NativeInterop/UnsafeNativeMethods.cs#L30